src/Http/Controllers/PasswordResetLinkController.php

```diff
@@ -32,13 +33,13 @@ public function create(Request $request): RequestPasswordResetLinkViewResponse
      */
     public function store(Request $request): Responsable
     {
-        $request->validate(['email' => 'required|email']);
+        $request->validate([Fortify::email() => 'required|email']);
 
         // We will send the password reset link to this user. Once we have attempted
         // to send the link, we will examine the response then see the message we
         // need to show to the user. Finally, we'll send out a proper response.
         $status = $this->broker()->sendResetLink(
-            $request->only('email')
+            $request->only(Fortify::email())
```

Comment:
Isn't this the only line that needs to change in this entire PR? Your database doesn't care what the POST variable is. Only what the column in the database is.